README.md

```diff
@@ -2,21 +2,25 @@
 
 Note: If you're looking for the Blockstack CLI repo it was merged with [Blockstack Core](https://github.com/blockstack/blockstack-core).
 
--   [Installation](#installation)
--   [About](#about)
--   [Auth](https://blockstack.github.io/blockstack.js/index.html#authentication)
--   [Profiles](https://blockstack.github.io/blockstack.js/index.html#profiles)
--   [Storage](https://blockstack.github.io/blockstack.js/index.html#storage)
--   [Documentation](#documentation)
--   [Compatibility](#compatibility)
--   [Contributing](#contributing)
--   [Maintainer](#maintainer)
--   [Testing](#testing)
--   [Releasing](#releasing)
+- [blockstack.js ![CircleCI](https://circleci.com/gh/blockstack/blockstack.js/tree/master) ![npm](https://www.npmjs.com/package/blockstack) ![npm](https://www.npmjs.com/package/blockstack) ![npm](https://www.npmjs.com/package/blockstack) ![Slack](http://slack.blockstack.org/)](#blockstackjs-circlecihttpscirclecicomghblockstackblockstackjstreemaster-npmhttpswwwnpmjscompackageblockstack-npmhttpswwwnpmjscompackageblockstack-npmhttpswwwnpmjscompackageblockstack-slackhttpslackblockstackorg)
+  - [Installation](#installation)
+  - [About](#about)
+  - [Documentation](#documentation)
+  - [Compatibility](#compatibility)
+  - [Contributing](#contributing)
+  - [Maintainer](#maintainer)
+  - [Testing](#testing)
+    - [Testing in a browser](#testing-in-a-browser)
+  - [Releasing](#releasing)
 
 ## Installation
 
-    $ npm install blockstack
+```
+$ npm install blockstack
+```
+
+You can import `blockstack.js` as a script without using a package manager. To securely use the latest distribution of blockstack.js from a CDN, add [this script declaration](mdincludes/include:script-dist-file.md) in your application. 
```

Comment:
I don't think link this works with github markdown `mdincludes/include:script-dist-file.md`

Reply:
I think I need  the full path there -- it won't resolve until the master merge in that case